vuln/npm/473.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": 473,
+  "title": "Code Injection in morgan",
+  "overview": "Code Injection Vulnerability in morgan Package",
+  "created_at": "2018-08-06",
+  "updated_at": "2018-09-28",
+  "publish_date": "1970-01-01",
+  "author": {
+    "name": "Cristian-Alexandru Staicu",
+    "website": "https://semmle.com/security",
+    "username": "cris_semmle"
+  },
+  "module_name": "morgan",
+  "cves": [],
+  "vulnerable_versions": "<1.9.1",
+  "patched_versions": ">=1.9.1",
+  "recommendation": "Update Other module module to version >=1.9.1",
```

Comment:
«Other module»?
Probably the json generator uses the wrong fild to pull the data from.